shared/xccdf/system/auditing.xml

```diff
@@ -100,7 +100,30 @@ the process, which in this case, is <tt>exe="/usr/sbin/httpd"</tt>.
 </li></ul>
 </description>
 
-<Rule id="service_auditd_enabled" severity="high" prodtype="rhel7">
+<Rule id="package_auditd_installed" severity="high" prodtype="debian8,debian9,ubuntu1404">
+<title>Install auditd Service</title>
+<description>The <tt>auditd</tt> service is an essential userspace component of
+the Linux Auditing System, as it is responsible for writing audit records to
+disk.
+</description>
+<ocil clause="the package is not installed">
+<package-check-macro package="auditd" />
+</ocil>
+<rationale>Without establishing what type of events occurred, it would be difficult
+to establish, correlate, and investigate the events leading up to an outage or attack.
+Ensuring the <tt>auditd</tt> service is active ensures audit records
+generated by the kernel are appropriately recorded.
+<br /><br />
+Additionally, a properly configured audit subsystem ensures that actions of 
+individual system users can be uniquely traced to those users so they
+can be held accountable for their actions.
+</rationale>
+<platform idref="cpe:/a:machine" />
+<oval id="package_auditd_installed" />
+<ref anssi="NT28(R50)" nist="AU-3,AC-17(1),AU-1(b),AU-10,AU-12(a),AU-12(c),AU-14(1),IR-5" disa="126,131" ossrg="SRG-OS-000038-GPOS-00016,SRG-OS-000039-GPOS-00017,SRG-OS-000042-GPOS-00021,SRG-OS-000254-GPOS-00095,SRG-OS-000255-GPOS-00096" pcidss="Req-10" cis="5.2.2" cjis="5.4.1.1" cui="3.3.1,3.3.2,3.3.6" />
```

Comment:
Do all these references line up to match that the `auditd` package should be installed?